opencog/moses/main/table-problems.cc

```diff
@@ -428,7 +428,8 @@ void pre_conj_table_problem::run(option_base* ob)
     // indexed number of rows.
     OC_ASSERT(not pms.meta_params.do_boosting,
         "Boosting not supported for the pre problem!");
-    REGRESSION(ctable, precision_conj_bscore,
+	type_node t_output_type = id::boolean_type;
```

Comment:
Now there's a mixture of tab and space indentation. Could you set everything in (smart) tab indentation in another PR?

Reply:
And for `table-problems.h` as well, just for consistency.